tests.py

```diff
@@ -81,5 +104,33 @@ def test_subclauses_arithmetic_booleans(self):
         self.run_vsm('subclauses_arithmetic_booleans', input_data,
                 expected_output, False)
 
+
+class TestIPC(TestVSM):
+
+    def setUp(self):
+        load_plugin('ipc.{}'.format(IPC_MODULE))
+
+    def test_simple0(self):
+        input_data = [('transmission_gear', '"reverse"')]
+        expected_output = ('car.backup', 'True')
```

Comment:
Same here: you can build the strings back from the output, I think that would be better.